### ISSUE TITLE
Launchpad: Gate the new Launchpad task list by blog IDs

### ISSUE THREAD

**Issue by valterlorran** [Automattic/wp-calypso#77271]
For starters, we want to display the new task lists, specifically the Customer Home Launchpad, only for new sites to avoid incompatibility with the current checklist on the Customer Home.

We should have a logic to enable the Launchpad on the customer's home to only new sites. Cutting off the old sites to prevent incompatibility. The incompatibility will be addressed in the future.

This implementation relies on having the `is_enabled` callback for the task list that https://github.com/Automattic/wp-calypso/issues/77270 should introduce.

Depends on: https://github.com/Automattic/wp-calypso/issues/77270
More about this discussion here: p1684324340423669/1684262724.806499-slack-C0Q664T29

**Comment by valterlorran** [comment]
We can follow the same reasoning that I described here: https://github.com/Automattic/wp-calypso/issues/77273#issuecomment-1576277653

**Comment by sixhours** [comment]
Adding the blog_id gate with the Automattician check in D113526-code

**Comment by sixhours** [comment]
Closed by D113526-code
